packages/devtools/src/constant.ts

```diff
@@ -39,5 +39,10 @@ export const defaultTabOptions: NuxtDevToolsOptions = {
   serverRoutes: {
     selectedRoute: null,
     view: 'tree',
+    globalInputs: {
```

Comment:
It seems this default object isn't merged deep, and causing `undefined` error when first access

Reply:
Also, maybe name it `inputDefaults` than `globalInputs`?